tests/qat/model/test_device.py

```diff
@@ -0,0 +1,2 @@
+class TestDevicesValidation:
+    pass
```

Comment:
I'll add some appropriate tests here once we all agree with the proposed structure of the `QuantumHardwareModel`.